src/CommunityToolkit/Xamarin.CommunityToolkit/Views/RangeSlider/SegmentedView/SegmentMode.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+
+namespace Xamarin.CommunityToolkit.UI.Views
+{
+	public enum SegmentMode
```

Comment:
I think it would be great if we could mix images and texts within the same segments control. Is there a reason why we have to distinguish between those two?